examples/layer-browser/src/render-metrics.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+const kB = 1024;
+const MB = 1024 * 1024;
+const GB = 1024 * 1024 * 1024;
+
+export function RenderMetrics(props) {
+  const metrics = props.metrics;
+
+  if (!metrics) {
+    return null;
+  }
+
+  return (
+    <div>
+      <div>FPS: {Math.round(metrics.fps)}</div>
+      <div>
+        GPU Frame Time: {metrics.gpuTimePerFrame.toFixed(2)}
+        ms
+      </div>
+      <div>
+        CPU Frame Time: {metrics.cpuTimePerFrame.toFixed(2)}
+        ms
+      </div>
+      <div>GPU Memory: {formatMemory(metrics.gpuMemory)}</div>
+    </div>
+  );
+}
+
+function formatMemory(mem) {
```

Comment:
It believe we already have this type of formatting in our `StatsWidget`?

I understand we wanted to present a more controlled interface with metrics so maybe we can't just use the underlying stats object functionalituy?

Reply:
As it currently stands, it would be messy to use the `StatsWidget`, because deck's metrics aren't kept in a `Stats` object. As we consider updating probe.gl's stats API, though, I think this would be a good use case to target.